views/github-configuration.hbs

```diff
@@ -28,98 +28,100 @@
       name='clientKey'
       value='{{clientKey}}'
     />
+
     <section class='getConfiguration'>
       <div class="headerImage">
         <img src='/public/assets/jira-and-github.png' alt="Jira and GitHub logos" />
       </div>
-      </p>
+
       <h2 class='f2 text-center text-normal getConfiguration__header'>Connect a GitHub organization to your Jira site</h2>
       <p class='jiraInstance'>{{jiraHost}}</p>
 
-      <div class='Box'>
-        <div class='getConfiguration__wrapper'>
-          <div class='tableHeader'>
-            <p
-              class='tableHeader__label org__cell'
-            >GitHub Organization</p>
-            <p class='tableHeader__label org__cell'>Repository
-              access</p>
-          </div>
+        <div class='Box'>
+          <div class='getConfiguration__wrapper'>
+            <div class='tableHeader'>
+              <p class='tableHeader__label org__cell'>GitHub Organization</p>
+              <p class='tableHeader__label org__cell'>Repository access</p>
+            </div>
 
-          <div class="githubOrgs__table">
-            {{#each installations}}
-              <div class='Box-row clearfix '>
-                {{!-- Organizations --}}
-                <div class='getConfiguration__orgContent org__cell'>
-                  <img
-                    class='avatar float-left mr-2'
-                    src='{{account.avatar_url}}'
-                    alt='Github organization avatar'
-                    height='25px'
-                    width='25px'
-                  />
-                  <span class='text-bold text-gray-dark'><a
-                      href='{{html_url}}'
-                    >{{account.login}}</a></span>
-                </div>
-
-                {{!-- Repository access --}}
-                <div class='getConfiguration__orgContent org__cell'>
-                  <p class="getConfiguration__orgContent__repoSelection">{{repoAccessType repository_selection}}</p>
-                  <p class="getConfiguration__orgContent__numberOfRepos">{{numberOfRepos}}</p>
-                  {{#if admin}}
-                    <a
-                      href='/github/subscriptions/{{id}}'
-                    >
+            <div class="githubOrgs__table">
+              {{#if installations}}
```

Comment:
Instead of having a large 'if' block, just do `#each installations` and the thing to show if there isn't installs, do `{{#unless installations}}` below :)

Reply:
This was something I didn't know existed! 🎉 Will clean that up now.